setup.py

```diff
@@ -7,7 +7,7 @@
 
 extras_require = {
     'tester': [
-        "eth-tester[py-evm]==0.1.0-beta.33",
+        "eth-tester[py-evm]==0.1.0-beta.35",
```

Comment:
```suggestion
        "eth-tester[py-evm]==0.1.0-beta.36",
```

Yet another devnum release today.